### DISCUSSION TITLE
[Bug]: Issue with findMany Query Filtering in AdminJS with Prisma

### DISCUSSION THREAD

**Discussion by mc-arenas** [prisma/prisma#21458] [Q&A]
### Bug description

Hello,

I'm facing an issue while trying to filter records using a findMany query in an AdminJS action handler. I'm using Prisma as my ORM. Here's a snippet of my code:

```js
const {userId} = request.payload ? request.payload['userId'].trim() : '0';
const transactions = await prisma.legacyTransaction.findMany({
  where: {
    user_id: Number(userId)
  }
});
```

When I run this code, I encounter the following error: PrismaClientValidationError: Argument equals is missing. I tried modifying the query to include equals like so:

```js
const transactions = await prisma.legacyTransaction.findMany({
  where: {
    user_id: {
      equals: Number(userId)
    }
  }
});
```
However, the error persists. The user_id field is defined as Int in my schema.prisma. I'm wondering if there's something with AdminJS that might be causing this issue or if it's a Prisma-related matter. Any insights or suggestions would be greatly appreciated.

Thank you!

### How to reproduce

execute query

### Prisma information

```prisma
// Add your schema.prisma
model LegacyTransaction {
  id                  Int       @id @default(autoincrement()) @db.UnsignedInt
  created             DateTime? @default(now())
  updated             DateTime? @default(now()) @updatedAt
  user_id             Int       @db.UnsignedInt
  type                String?   @db.VarChar(190)
  origin              String?
  o_amount            Decimal?  @db.Decimal(36, 18)
  o_address           String?
  destination         String?
  d_amount            Decimal?  @db.Decimal(36, 18)
  d_address           String?
  status              String?   @db.VarChar(12)
 .....

  LegacyUser LegacyUser @relation(fields: [user_id], references: [id])

  TradeTransaction LegacyTradeTransaction[]

  @@index([type, tx_hash])
  @@map("transaction")
}ts
// Add your code using Prisma Client
```
 ```
const transactions = await prisma.legacyTransaction.findFirst({
              where: {
                user_id: {
                    equals: Number(userId)
                }
            }
            });
```

### Environment & setup

- OS: macOS
- Database: MySQL
- Node.js version: v18.16.1

### Prisma Version

```
prisma                  : 5.3.1
@prisma/client          : Not found
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine 61e140623197a131c2a6189271ffee05a7aa9a59 (at ../../../Library/pnpm/global/5/.pnpm/@prisma+engines@5.3.1/node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 61e140623197a131c2a6189271ffee05a7aa9a59 (at ../../../Library/pnpm/global/5/.pnpm/@prisma+engines@5.3.1/node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.3.1-2.61e140623197a131c2a6189271ffee05a7aa9a59
Default Engines Hash    : 61e140623197a131c2a6189271ffee05a7aa9a59
Studio                  : 0.494.0
```

**Comment by mc-arenas** [comment]
yes it was that, I just saw it after I put the issue and I forgot to close it, thanks!

**Reply by ludralph** [reply]
Hi @mc-arenas 👋 

Has this issue been resolved based on your last comment?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
